tests/unit/expected_test_results/additives/fr-vitamine-c-with-function.json

```diff
@@ -1,17 +1,15 @@
 {
-   "additives_n" : 1,
-   "additives_original_tags" : [
-      "en:e300"
-   ],
-   "additives_tags" : [
-      "en:e300"
-   ],
+   "additives_n" : 0,
+   "additives_original_tags" : [],
+   "additives_tags" : [],
    "amino_acids_tags" : [],
    "ingredients_lc" : "fr",
    "ingredients_text" : "Eau, sucre, antioxydant: vitamine C",
```

Comment:
Here it is used as an additive

Reply:
understood. It was duplicates between additives.txt and vitamins.txt, the solution was to keep it only in vitamins.txt. Although we expect that "fr-vitamine-c-with-function" to produce different expect result :-(